### DISCUSSION TITLE
Do you have a blog related to Oqtane?

### DISCUSSION THREAD

**Discussion by thabaum** [oqtane/oqtane.framework#4216] [Q&A]
Wondering who has a blog relating to Oqtane?

I figured I would start a thread for community members to share a link with a plug about your blog here to help other members know about it as I know many like myself love to read and share them!

:rocket:

Example:
### Oqtane Framework Official Blog
**https://www.oqtane.org/Blog**

**Comment by vnetonline** [comment]
https://LearnOqtane.com Learn Oqtane Fast!

![logo-learn-oqtane-3d-transparent-black](https://github.com/oqtane/oqtane.framework/assets/15242923/b75c19e8-ff63-4357-a5d6-393440cdc241)
LEARN | BUILD | DEPLOY | MONETIZE

Video Courses and Progress Log are being updated as I type this!
